test/NinjaTools.FluentMockServer.Tests/TestHelpers/Mocks/MockHandler.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using NinjaTools.FluentMockServer.Models;
+using Xunit.Abstractions;
+
+namespace NinjaTools.FluentMockServer.Tests.TestHelpers.Mocks
+{
+    public class MockHandler : HttpMessageHandler
+    {
+        private readonly ITestOutputHelper _outputHelper;
+
+
+        public MockHandler(ITestOutputHelper outputHelper)
+        {
+            _outputHelper = outputHelper;
+        }
+
+
+        public int RequestCounter;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/app/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4722826)